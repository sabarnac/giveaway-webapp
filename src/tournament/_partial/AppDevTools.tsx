import AppDevTools from "./AppDevTools.react";

export default AppDevTools;
