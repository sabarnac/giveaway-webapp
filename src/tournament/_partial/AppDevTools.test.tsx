import React from "react";
import AppDevTools from "./AppDevTools";
import { create, ReactTestRenderer } from "react-test-renderer";

describe("AppDevTools Component.", () => {
  it("Matches snapshot.", () => {
    const component: ReactTestRenderer = create(<AppDevTools />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
