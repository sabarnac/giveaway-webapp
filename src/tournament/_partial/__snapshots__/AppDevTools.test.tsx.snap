// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppDevTools Component. Matches snapshot. 1`] = `null`;
